resources/resources.go

```diff
@@ -61,13 +61,40 @@ func (rp ResourcePlan) ApplyPlan(name string, vc VaultClient) (Resource, error)
 		ScaleOutCooldown:    ucd,
 		NomadToComputeRatio: rp.NomadToComputeRatio,
 		Name:                name,
+		nextResetTime:       time.Unix(1<<63-62135596801, 999999999),
```

Comment:
Put a link to show why this set of values are used.

Reply:
I see a repeat in the code below, would be good to assign this to a named constant and put the link comment there instead?

Reply:
I tried setting it as a constant but unfortunately it's not allowed (maybe because it's an expression?). Will see if i can work around that

Reply:
Doesn't seem like i can do this

Reply:
Okay it's because `const` for Golang is really for literal constants haha. We kind of just want a "readonly" value here, which it can't do.

The next best is to return this value under a function name that acts like constant. But otherwise if you don't prefer that, it's fine also, since there are only 2 of them. Just comment on the first usage is fine by me.